mllib/src/main/scala/org/apache/spark/mllib/classification/NaiveBayes.scala

```diff
@@ -272,12 +278,11 @@ class NaiveBayes private (
 
   /**
    * Set the model type using a string (case-sensitive).
-   * Supported options: "Multinomial" and "Bernoulli".
-   * (default: Multinomial)
+   * Supported options: "multinomial" (default) and "bernoulli".
    */
-  def setModelType(modelType:String): NaiveBayes = {
+  def setModelType(modelType: String): NaiveBayes = {
```

Comment:
I tried that option in the first version. Making it case-insensitive is easy but it may cause unexpected errors in the user code. 

~~~scala
nb.setModelType("Bernoulli")
if (nb.getModelType() == "Bernoulli") { // It becomes lowercase here.
  ...
}
~~~

I can keep the input untouched but call `toLowerCase every time we need it.